code/src/components/StartScreen.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+import game from '../reducers/game';
+import { startGame } from '../reducers/game';
+import Footer from 'components/Footer';
+
+const StartScreen = () => {
+	const [inputValue, setInputValue] = useState('');
+
+	const dispatch = useDispatch();
+
+	const onUsernameInput = () => {
+		dispatch(game.actions.setUsername(inputValue));
+		dispatch(startGame());
```

Comment:
This I think is where you could put a check for whether the inputValue is 'valid' and if not, render a little message that says to enter a valid username or something :)